rpc/src/v1/helpers/dapps.rs

```diff
@@ -22,12 +22,18 @@ use v1::types::LocalDapp;
 pub trait DappsService: Send + Sync + 'static {
 	/// List available local dapps.
 	fn list_dapps(&self) -> Vec<LocalDapp>;
+	/// Refresh local dapps list
+	fn refresh_local_dapps(&self) -> bool;
 }
 
+/// TODO: Update this trait to a struct
 impl<F> DappsService for F where
-	F: Fn() -> Vec<LocalDapp> + Send + Sync + 'static
+	F: Fn(bool) -> (Vec<LocalDapp>, bool) + Send + Sync + 'static
```

Comment:
https://github.com/paritytech/parity/blob/04bd4d9559e1f99259fcc41b5a5fd7e4e69f2c7e/parity/dapps.rs#L294

Yeah, I'll change it.